docs/Setup/Upgrade.md

```diff
@@ -1,67 +1,60 @@
-    web server and the command line user. See Symfony's [Setting up or Fixing File Permissions](http://symfony.com/doc/current/setup/file_permissions.html) 
+  - Ensure that the `/var/cache` and `/var/log` directories must be writable both by the web server and the command line user.
+    See Symfony's [Setting up or Fixing File Permissions](http://symfony.com/doc/current/setup/file_permissions.html) 
     to see potential solutions to this problem when installing from the CLI.
-  - Note 2: If you have `mod_suexec` installed for Apache the CLI will run into permission problems. (If you are not sure 
-    check your phpinfo.) `mod_suexec` often is used in shared hosting environments. In this case, the CLI installer is not 
-    recommended, please use the Web Installer.
-- Copy your previous installation's `/app/config/custom_parameters.yml` to `/config/services_custom.yaml` in your new installation.
-  - note the name change and the suffix changed from `.yml` to `.yaml`
+  - If you have `mod_suexec` installed for Apache the CLI will run into permission problems. (If you are not sure 
+    check your phpinfo.) `mod_suexec` is often used in shared hosting environments. In this case, the CLI installer is not 
+    recommended. Instead, please use the Web Installer.
+- Please note, your _web root_ **must** now point to the `public/` directory. Adjust your symlinks or directory structure as needed.
+  (see https://symfony.com/doc/current/setup/web_server_configuration.html for more information).
 
 ### Continue
 
-- Please note, the _web root_ **must** now point to the `public/` directory. Adjust your symlinks or directory structure as needed.
-  (see https://symfony.com/doc/current/setup/web_server_configuration.html for more information).
-- Copy your _compatible_ custom theme to your new installation. Themes are now placed in `src/extensions`.
-  - After upgrade, the site will default to the ZikulaBootstrapTheme until you complete testing of your custom theme.
-- Return _compatible_ extensions to the `src/extensions` directory.
-  - **DO NOT copy the old Profile and Legal module** as new versions of these are provided, and their location may differ.
-- Copy your backup contents of `/userdata` (1.x) or `/web/uploads` (2.x) into `/public/uploads`
+- Copy old files to _new_ locations:
+    - Copy your previous installation's `/app/config/custom_parameters.yml` to `/config/services_custom.yaml` in your new installation.
+        - note the name change and the suffix changed from `.yml` to `.yaml`
+    - Copy your _compatible_ custom theme to the `src/extensions` directory (There is no `/themes` directory any longer).
+        - After upgrade, the site will default to the ZikulaBootstrapTheme until you complete testing of your custom theme.
+    - Return _compatible_ modules (extensions) to the `src/extensions` directory.
+        - **DO NOT copy the old Profile and Legal module** as new versions of these are provided, and their location may differ.
```

Comment:
Maybe add: `If unsure about whether a module is compatible of whether a new compatible version is available please ask for support in our Slack channels.` ?